uaclient/entitlements/base.py

```diff
@@ -38,6 +41,9 @@ class UAEntitlement(metaclass=abc.ABCMeta):
     # Wheter that entitlement is in beta stage
     is_beta = False
 
+    # Help info message for the entitlement
+    _help_info = None  # type: str
```

Comment:
I think we can drop this as caching the value doesn't make sense given that this will only be used via a single cmdline call at the moment and this cache would be set up each time someone calls ua help <blah>